examples/java/build.gradle

```diff
@@ -109,13 +109,8 @@ dependencies {
   }
   directRunnerPreCommit project(path: ":runners:direct-java", configuration: "shadow")
   flinkRunnerPreCommit project(":runners:flink:${project.ext.latestFlinkVersion}")
-  // TODO: Make the netty version used configurable, we add netty-all 4.1.17.Final so it appears on the classpath
-  // before 4.1.8.Final defined by Apache Beam
-  sparkRunnerPreCommit "io.netty:netty-all:4.1.17.Final"
```

Comment:
Why it was removed as well? Spark2 specific?

Reply:
This is far obsolete and has no effect ...